stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -14,6 +14,13 @@ class AsyncResult():
     def ready(self) -> bool: ...
     def successful(self) -> bool: ...
 
+class Event():
```

Comment:
just `class Event:` is fine (or `class Event(object):` to prepare for an eventual unification with the Python 2 stubs)